src/pages/workspace/taxes/WorkspaceTaxesPage.tsx

```diff
@@ -129,23 +134,87 @@ function WorkspaceTaxesPage({policy, route}: WorkspaceTaxesPageProps) {
         </View>
     );
 
+    const deleteTaxes = useCallback(() => {
+        if (!route.params.policyID) {
```

Comment:
maybe extract `policyID` from routes in the function declaration? We use it here many times